k8s/production/runners/public/x86_64/v2/windows.yaml

```diff
@@ -0,0 +1,142 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: runner-x86-v2-pub-windows
+  namespace: gitlab
+spec:
+  interval: 10m
+  url: https://charts.gitlab.io
+
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: runner-x86-v2-pub-windows
+  namespace: gitlab
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: gitlab-runner
+      version: 0.54.0 # gitlab-runner@v16.1.0
+      sourceRef:
+        kind: HelmRepository
+        name: runner-x86-v2-pub-windows
+  values:
+    image:
+      image: gitlab-org/gitlab-runner
+      tag: alpine-v16.1.0
+    imagePullPolicy: IfNotPresent
+    replicas: 1
+
+    gitlabUrl: "https://gitlab.staging.spack.io/"
```

Comment:
Prod should not be staging. For initial deployment only deploy this to staging for development